test/success/ua.prl

```diff
@@ -0,0 +1,49 @@
+Def HasAllPaths (#C) = [(-> [c c' : #C] (path {_} #C c c'))].
+
+Def IsContr (#C) = [(* [c : #C] (HasAllPaths #C))].
+
+Def Fiber (#A;#B;#f;#b) = [(* [a : #A] (path {_} #B ($ #f a) #b))].
+
+Def IsEquiv (#A;#B;#f) = [(-> [b : #B] (IsContr (Fiber #A #B #f b)))].
+
+Def Equiv (#A;#B) = [(* [f : (-> #A #B)] (IsEquiv #A #B f))].
+
+Def Id = [(lam [a] a)].
+
+Thm IdIsEquiv (#A) : [
+  #A hcom type
+  >>
+  (IsEquiv #A #A Id)
+] by [
+  unfold IsEquiv;
+  fresh a -> refine dfun/intro;
+  [ unfold IsContr;
+    [ refine record/intro;
+      [ `(tuple ,a (abs {_} ,a))
+      , unfold HasAllPaths;
+        lam c, c'. <i>
+	  `(tuple
+	    (hcom{1~>0} #A ,a [i=0 {j} (@ (! proj2 ,c) j)] [i=1 {j} (@ (! proj2 ,c') j)])
+	    (abs {j} (hcom{1~>j} #A ,a [i=0 {j} (@ (! proj2 ,c) j)] [i=1 {j} (@ (! proj2 ,c') j)])))
+      , auto
+      ]
+    ]
+  , auto
+  ]
+].
```

Comment:
This is pretty cool!!

By the way, a higher-level/programming-style version of this script could be the following:

```

Thm IdIsEquiv (#A) : [
  #A hcom type
  >>
  (IsEquiv #A #A Id)
] by [
  unfold IsEquiv; unfold IsContr; unfold Fiber; unfold HasAllPaths;
  lam a.
  < proj1 =
     <proj1 = use a,
      proj2 = <_> use a>
  , proj2 =
     lam <proj2 = c>, <proj2 = c'>. <i>
       < proj1 =
          `(hcom{1~>0} #A ,a
            [i=0 {j} (@ ,c j)]
            [i=1 {j} (@ ,c' j)])
       , proj2 =
           <j> `(hcom{1~>j} #A ,a
                 [i=0 {j} (@ ,c j)]
                 [i=1 {j} (@ ,c' j)])
       >
  >
].
```

Reply:
Can we make `unfold` take multiple arguments, by the way?

Reply:
@favonia I was just grimacing over that myself 😄 I think we should definitely do that.